### ISSUE TITLE
[FUNCTIONALITY]: Sortable Table - SHOULD use our base Table component

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/vets-design-system-documentation#229]
@1Copenut commented on [Mon May 18 2020](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9197)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️  Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our [sortable table component](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/sortabletable/) ~~would benefit greatly from a basic `<Table />` component~~ should be a prop-driven extension on top of our [Table component](https://github.com/department-of-veterans-affairs/component-library/blob/master/src/components/Table/Table.jsx). This way we could build out more complex tables as extensions than completely new components. Thinking:

* Tables with multiple heading rows
* Tables with rows or columns that span multiple
* Tables with sortable columns

## Related Issues
* #9193 
* #9194 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria
- [ ] HTML validates with W3C validator or other HTML5 checker
- [ ] No axe errors
- [ ] [Table component](https://github.com/department-of-veterans-affairs/component-library/blob/master/src/components/Table/Table.jsx) is used to create extended or enhanced table components
- [ ] Visual styling does not change, except where noted by issue tickets

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/sortabletable/

**Comment by bkjohnson** [comment]
Closing since our `<va-table>` component [has sorting functionality driven by props](https://github.com/department-of-veterans-affairs/component-library/pull/318).
